src/controllers/accountController.ts

```diff
@@ -280,6 +280,7 @@ export const isEmailAvailable = async (email: string): Promise<boolean> => {
   const [inDatabase, inPending] = await Promise.all([
```

Comment:
I realize this code needs slightly more tweaking - needs to capture the results in three different variables
```suggestion
  const [inDatabase, inPending, inNeedsVerify] = await Promise.all([
```